modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -308,13 +324,23 @@ public List<ObjectRelationship> getObjectRelationships(
 			objectDefinitionId1, objectDefinition2, type);
 	}
 
-	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ObjectRelationship updateObjectRelationship(
 			long objectRelationshipId, String deletionType,
 			Map<Locale, String> labelMap)
 		throws PortalException {
 
+		return updateObjectRelationship(
+			objectRelationshipId, deletionType, labelMap, null);
+	}
```

Comment:
Do we really need to keep this method?

Reply:
As discussed offline, I'll follow previous precedent.